File: miscellaneous/jan1.py
import re

def findWholeWord(w):
    return re.compile(r'\b({0})\b'.format(w), flags=re.IGNORECASE).search


t=int(input())
for _ in range(t):
    str=input()
    if(findWholeWord('not')(str)):
        print("Real Fancy")
    else:
        print("regularly fancy")

File: miscellaneous/jan1.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        string str;
        getline(cin,str);
        int res=0;
        istringstream ss(str);
        do {
            string word;
            ss >> word;
            cout<<word<<endl;
            if(word=="not")
            {
                res=1;
                break;
            }
        } while (ss);

        if(res)
            cout<<"Real Fancy\n";
        else
            cout<<"regularly fancy\n";
    }
}
